### DISCUSSION TITLE
Attach/Detach `device_tracker` to `person`

### DISCUSSION THREAD

**Discussion by jlpouffier** [frenck/spook#325] [Ideas]
**Context**
I have a smart(ish) car.
My car also acts as a device tracker in Home Assistant.

The update density of the car location is extremely good. I get at least one update per second.
![image](https://github.com/frenck/spook/assets/5878296/77cf49ef-bbec-43ee-88e6-b68f3db4b872)

On the other hand, my phone does not update that fast.
I usually have to wait quite a while to see an update.
![image](https://github.com/frenck/spook/assets/5878296/2d3ae750-8ca7-4b6c-9b0f-0ee6f1162c5d)

I can detect easily who's the driver of the car (Either with a sensor on Android, or a personal siri shortcut automation on iOs)

**Idea**
New services to be able to Attach and Detach a `device_tracker` from a `person`.

So that:
- When I detect I am in my car, my car becomes my very powerful and overkill `device_tracker`
- When I detect I am not in my car anymore, my phone becomes my lame `device_tracker`

**Comment by frenck** [comment]
Nice, that is doable 👍

**Comment by frenck** [comment]
This is being implemented in: https://github.com/frenck/spook/pull/331

**Comment by frenck** [comment]
Released in: <https://github.com/frenck/spook/releases/tag/v1.2.0>
